setup.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+setup(name='defectMOF',
+      version='0.1.0',
+      description='Introducing Missing-linker Defects in Metal-Organic Frameworks',
+      author='Meiirbek Islamov',
+      author_email='mei12@pitt.edu',
+      url='https://github.com/meiirbek-islamov/defects',
+      packages=['defectMOF'],
```

Comment:
packages should not have caps; this should probably be defectmof. I think you added the directory as just `defects` so that is your package.